src/Sylius/Bundle/CoreBundle/Fixture/Factory/ProductExampleFactory.php

```diff
@@ -228,6 +228,9 @@ protected function configureOptions(OptionsResolver $resolver): void
 
             ->setDefault('tax_category', null)
             ->setAllowedTypes('tax_category', ['string', 'null', TaxCategoryInterface::class])
+
+            ->setDefault('price', null)
+            ->setAllowedTypes('price', ['int', 'null'])
```

Comment:
As an improvement it's 👌, but we should probably aim to pass multiple prices for multiple channels

Reply:
Supporting something like this would be very nice:

- Different price for every channel
```
prices:
    FASHION_WEB: 10.00
    ANOTHER_CHANNEL: 20.00
```

- Same price for all channels
```
prices: 10.00
```

Reply:
You are right, pass multiple prices for multiple channels will be good solution 👌